src/js/structs/Job.js

```diff
@@ -0,0 +1,21 @@
+import Item from './Item';
+
+module.exports = class Job extends Item {
+  constructor() {
+    super(...arguments);
+
+    const id = this.getId();
+    if (id !== '/' && (!id.startsWith('/') || id.endsWith('/'))) {
+      throw new Error(`Id (${id}) must start with a leading slash ("/") and should not end with a slash, except for root id which is only a slash`);
```

Comment:
@orlandohohmeier @mlunoe 
Yeah, cool, doing it the ChronosStore is a good idea imho! Is that also the way it could work with the Services?

BTW, never trust the backend! ;)